lib/zenaton/services/graphql.rb

```diff
@@ -0,0 +1,160 @@
+# frozen_string_literal: true
+
+require 'net/http'
+require 'zenaton/exceptions'
+
+module Zenaton
+  # Collection of utility classes for the Zenaton library
+  module Services
+    # Wrapper class around HTTParty that:
```

Comment:
HTTParty is no longer a dependency. I guess the docs from the http service need to be updated too.